src/jace/optimization.py

```diff
@@ -19,7 +15,7 @@
 
 
 if TYPE_CHECKING:
-    from jace import translator
+    import jace
```

Comment:
I understand han and i changed that.
For jace model I now settled in case of alias, I now have made some scheme, first if the module has multiple words then the first letter of each is used and the last one.
This is not nice but it is deterministic, at least in some way.
I looked but I think it should all changed.

Note that the alias of external packages the scheme is `${PKGNAME}_${SOME_GOOD_NAME}`

Reply:
If those conventions are expected to be followed by anyone contributing to this project, then they should be added do the coding conventions.